src/ipc/ipc4/dai.c

```diff
@@ -202,19 +154,18 @@ void dai_dma_release(struct comp_dev *dev)
 	/* put the allocated DMA channel first */
 	if (dd->chan) {
 		struct ipc4_llp_reading_slot slot;
-		uint32_t llp_reg_offset;
-		uint32_t node_id;
+		struct llp_slot_info *llp;
```

Comment:
Same here, is this needed? You can probably just dereference with
`add->slot_info.node_id` ?

Reply:
refined, thanks